pandas/core/arrays/base.py

```diff
@@ -1073,6 +1073,9 @@ def _reduce(self, name, skipna=True, **kwargs):
         """
         raise TypeError(f"cannot perform {name} with type {self.dtype}")
 
+    def __hash__(self):
```

Comment:
may want to add some text to the top of the EA that says EA by default are not hashable